app/src/main/java/campus/tech/kakao/map/SavedSearchAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package campus.tech.kakao.map
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class SavedSearchAdapter(
+    private val context: Context,
+    private var data: List<String>,
+    private val onItemClick: (String) -> Unit
+) : RecyclerView.Adapter<SavedSearchAdapter.SavedSearchViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SavedSearchViewHolder {
+        val view = LayoutInflater.from(context).inflate(R.layout.activity_item_view, parent, false)
```

Comment:
context 가 LayoutInflater 를 위해서 사용되는 것으로 보여요. parent 의 context 를 사용한다면 Adapter 가 context 를 파라미터로 받지 않아도 됩니다.
```suggestion
        val view = LayoutInflater.from(parent.context).inflate(R.layout.activity_item_view, parent, false)
```